File: angular-new-app/ClientApp/src/app/medlems/medlem.ts

type Guid = string & { isGuid: true};
export interface Medlem {
  medlem_Id: Guid;
  fornavn: string;
  etternavn: string;
  bosted?: string;
  mobilTlf: number;  
  email: string;
  fodselsdato: Date;  
  currentKontiId?: number;
}   

     

File: angular-new-app/Models/Medlem.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace angular_new_app.Models
{
    public partial class Medlem
    {
        [Key]
        public Guid Medlem_Id { get; set; }      
	public string Fornavn { get; set; }
        public string Etternavn { get; set; }
	public string? Bosted { get; set; }
        public int MobilTlf { get; set; }
	public string Email { get; set; }
	[Column(TypeName="date")]
	public DateTime Fodselsdato { get; set; }
        public int CurrentKontintId { get; set; }
	public Kontingent Kontingent { get; set; }
    }
}
